fix: [AARD-2113] Wiring Panel Styling Fixes

HunterBarclay opened the pull request:
## Task
AARD-2113

Julian broke my wiring panel. I fix

## Problem
Styling issues since UI refactor for the wiring panel.

<img width="1800" height="1169" alt="image" src="https://github.com/user-attachments/assets/928acf03-a2b4-4c94-82bd-6973e2975530" />

## Verification
<img width="1800" height="1169" alt="Screenshot 2025-11-26 at 11 18 23 PM" src="https://github.com/user-attachments/assets/7e4d0984-09bc-4602-b6c9-1e66e61dfc03" />
<img width="1800" height="1169" alt="Screenshot 2025-11-26 at 11 18 06 PM" src="https://github.com/user-attachments/assets/3eba3fc7-a5ae-491f-9a1b-b502ddca9ef8" />

---

Before merging, ensure the following criteria are met:

- [ ] All acceptance criteria outlined in the ticket are met.
- [ ] Necessary test cases have been added and updated.
- [ ] A feature toggle or safe disable path has been added (if applicable).
- [ ] User-facing polish:
  - Ask: *"Is this ready-looking?"*
- [ ] Cross-linking between Jira and GitHub:
  - PR links to the relevant Jira issue.
  - Jira ticket has a comment referencing this PR.

HunterBarclay committed:
Fix wiring panel

PepperLola reviewed (changes requested):
When I run the regular Java simulation sample and load Dozer, it doesn't immediately display the simulation inputs and outputs. I can refresh the Robot IO panel and that will sometimes make them load properly. Initially it took multiple refreshes to get it to work but I haven't been able to reproduce that since. The refresh/reset button at the top removes all the simulation inputs and outputs and makes another Robot IO panel refresh necessary.

<img height="512" alt="image" src="https://github.com/user-attachments/assets/0411e018-6dc1-4e52-8ee5-aba5b09c4af9" />

When I add a junction node it shows no inputs or outputs until I refresh from the Robot IO panel.

<img height="512" alt="image" src="https://github.com/user-attachments/assets/c1e21289-d9df-4f34-ae3f-00af9fdbef04" />

HunterBarclay commented:
@PepperLola Odd. Java is borked on my laptop right now and I haven't been able to test fully with robot code yet. Let me know if you can see any issues appearing in the console.

HunterBarclay commented on fission/src/ui/panels/simulation/WiringPanel.tsx:
```diff
@@ -263,14 +286,14 @@ const WiringComponent: React.FC<ConfigComponentProps> = ({ setConfigState, simCo
     const [nodes, setNodes, onNodesChange] = useNodesState([] as FlowNode[])
     const [edges, setEdges, onEdgesChange] = useEdgesState([] as FlowEdge[])
-    const [_refreshHook, refreshGraph] = useReducer(x => !x, false) // Whenever I use reducers, it's always sketch. -Hunter
+    const [refreshHook, refreshGraph] = useReducer(x => !x, false) // Whenever I use reducers, it's always sketch. -Hunter
```
@PepperLola I've been using this as a janky way to refresh the graph. Probably need to re-evaluate.

HunterBarclay committed:
Fix UI issues with ReactFlow graph and checkboxes.

!: Auto Tester doesn't work and it appears the robot can't be controlled
via robot code at the moment...

Dhruv-0-Arora commented on fission/src/ui/panels/simulation/WiringPanel.tsx:
```diff
@@ -38,6 +39,11 @@ import FlowInfo from "@/ui/components/simulation/FlowInfo"
 import WiringNode from "./WiringNode"
 
+/**
+ * WARNING: Please test *thoroughly* when making changes. React Flow is very tempermental with how nodes
```
```suggestion
 * WARNING: Please test *thoroughly* when making changes. React Flow is very temperamental with how nodes
```

Dhruv-0-Arora commented on fission/src/ui/panels/simulation/WiringPanel.tsx:
```diff
@@ -1,4 +1,5 @@
-import { Grid, Stack } from "@mui/material"
+import "@xyflow/react/dist/style.css"
```
nice package
